cla_backend/apps/call_centre/management/commands/create_and_assign_agilisys_organisation.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management import BaseCommand
+from call_centre.models import Organisation, Operator
+
+
+class Command(BaseCommand):
+    help = "Create Agilisys organisation and assign current agilisys operators to that organisation"
+
+    def handle(self, *args, **options):
+        (organisation, created) = Organisation.objects.get_or_create(name="Agilisys")
```

Comment:
This is fine and valid, but I'm not used to seeing the left hand side written with the parentheses
More normal to write `organisation, created = ...` I think